Libbna/CUSTOM-CMS
# 169: Added Comment Section using disqus

### Completed

**Testing Steps:**

- Go to any article page.
- Scroll down to the bottom.
- check if the comment box appears or not.
- try commenting.
- Refer to the following screenshot.

![image](https://user-images.githubusercontent.com/81369580/151380751-4591c61a-131b-4df9-af36-31069bbf94b8.png)
